### DISCUSSION TITLE
[BUG] Status badge has wrong width when workflow name contains Unicode 🦄

### DISCUSSION THREAD

**Discussion by LinqLover** [community/community#26772] [Actions]
<p>I have already reported this issue here (<a href="https://github.com/actions/toolkit/issues/742" rel="noopener nofollow ugc">actions/toolkit#742</a>), so allow me to copy the report over from there:</p>
<blockquote>
<p><strong>Describe the bug</strong><br>
If a workflow name contains Unicode characters, the width for the status badge is too small:</p>
<p><img src="https://user-images.githubusercontent.com/38782922/181099837-7f0c2c63-776b-4e76-a636-0d4861375d7c.png" alt="image" data-base62-sha1="8dvdJIg3ETFJJJbpIQxJe4bWYlN" width="146" height="34"><br>
Example build:<br>
<a href="https://github.com/hpi-swa-lab/SqueakByExample-english/actions/runs/665402056" class="inline-onebox" rel="noopener nofollow ugc">Turn off notifications unless build was scheduled · hpi-swa-lab/SqueakByExample-english@6128824 · GitHub</a></p>
</blockquote>
<p>Thanks in advance for fixing it! :slight_smile:</p>

**Comment by Simran-B** [comment]
<p>These forums are actually not a good place for feature requests and bug reports. You should use the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="noopener nofollow ugc">feedback form</a> instead.</p>

**Comment by LinqLover** [comment, marked as answer]
<p>Sorry, <a class="mention" href="/thboop">@thboop</a> had guided me here. I have just submitted a feedback issue.</p>

**Comment by Simran-B** [comment]
<p>Yeah, I also saw that for other requests. This is a place for discussions, though. It makes the life of GitHub support and engineering a lot easier if feature requests and bug reports are submitted via the form, as that allows them to track these issues much better. Here, these things can get easily overlooked.</p>

**Comment by LinqLover** [comment]
<p>It’s just a bit sad for an open-source company not to have a public issue tracker but private feedback forms, but that’s of course not your fault … :slight_smile:</p>

**Comment by Simran-B** [comment]
<p>Well, there’s the public roadmap, but it appears to be reserved for larger projects. It would be nice to have another project board with the small scale things including a comment what product management thinks about it.</p>

**Comment by LinqLover** [comment]
<p>Most important for me is traceability. I would like to get updates if my - or someone else’s - issue has been solved, so they would need to pin our issues to their project board …</p>

**Comment by Simran-B** [comment]
<p>Yeah, I had the same thought. If you see that someone already submitted a request or bug, then you want to be able to enable notifications to get informed about any progress or the resolution. I started a discussion with the community management and Rising Stars (the most active community members) about this, let’s see where it goes.</p>

**Comment by LinqLover** [comment]
<p>In case someone is needing a workaround, <a href="http://shields.io" rel="noopener nofollow ugc">shields.io</a> is also able to generate badges for every single workflow.</p>

**Comment by theory** [comment]
<p>Glad to find this post; I’ve submitted a note via the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="noopener nofollow ugc">feedback form</a>, as well. This just doesn’t look great.</p>
<p><img src="https://user-images.githubusercontent.com/46604/181099846-56b3c59c-5b6c-4890-8982-50dc56ffdcbc.png" alt="Screen Shot 2021-11-06 at 17.35.59" data-base62-sha1="ffrU9YizwI2VTYDmdkFV6xxCr2f" width="236" height="62"></p>
